.github/ISSUE_TEMPLATE/website.md

```
@@ -0,0 +1,26 @@
+---
+name: 🌐 Website issue
+about: Issues related to website.
+title: '[Website]'
+labels: 'area/website, framework/jekyll'
+assignees: ''
+---
+
+### Current Behavior
```

markdownlint-cli2 (0.23.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)